### PR TITLE
GameRoom player list & GameLobby basic styling

### PR DESCRIPTION
**Pull request by bhoconnor**
## Description

- Maps players into a bullet point list in GameRoom, with some very basic inline styling
- Adds some basic styling into the bullet point list in GameLobby
- Center aligns join room input field in GameLobby

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

Sorry no before

### After

**GameRoom changes**: 

image

and

**GameLobby changes**: 

image

### CODE REVIEW

**Comment by bhoconnor** [comment]
Note that the above "[Hide profile if not logged in](https://github.com/Code-the-Dream-School/bb-practicum-team-2-front/pull/34/commits/64ba4accdab35c7a6abbba2e9d85faeb5ce27c33)" commit was undone by the later "[Bullet point current GameRoom players & undo hamburger nav changes](https://github.com/Code-the-Dream-School/bb-practicum-team-2-front/pull/34/commits/7b413b5f5833c188b157905a489e0377874c82ff)" commit. 

This branch also has all of the latest `dev` branch incorporated in it (with help from @benochi tonight, hat tip to him).

File: src/components/GameRoom.js [hunk 1]
```diff
@@ -78,50 +75,54 @@ function GameRoom({ room, setInRoom, userName, host, gamesWon,_id }) {
...
-          headers: {
-            "Content-Type": "application/json",
-            Authorization: `Bearer ${token}`,
-          },
-          body: JSON.stringify({
-            gamesWon: ++gamesWon
-          })
-        });
+        const response = await fetch(
+          `http://localhost:4000/api/v1/user/${_id}`,
```
Comment (frankstepanski) [inline src/components/GameRoom.js]: Can't use hard-coded server URLs anymore. Remember, we are deployed now. Need to use the base_url variable.

**Review by frankstepanski — CHANGES_REQUESTED** [review]
Need to change the hard-code localhost:4000 URL. We are deployed now. We use the base_url variable now.

File: src/components/GameRoom.js [hunk 2]
```diff
@@ -174,20 +171,42 @@ function GameRoom({ room, setInRoom, userName, host, gamesWon,_id }) {
...
+                }}
+              >
+                - {player}
+              </li>
+            </div>
+          ))}
+        </ul>
+        {console.log("players", players)}
+        <button onClick={leaveRoom}>Leave Room</button>
+        <button onClick={disconnectRoom}>Disconnect</button>
```
Comment (mautoranzo) [inline src/components/GameRoom.js:197]: The button disconnect is not necessary any more, now we have the Logout btn (line 167 this same file)

**Review by mautoranzo — CHANGES_REQUESTED** [review]

File: src/components/GameRoom.js (hunk 1 above)
Comment (frankstepanski) [inline src/components/GameRoom.js]: Can't have hard-coded URL to the server.

```
try {
        const response = await fetch(
          `http://localhost:4000/api/v1/user/${_id}`,
```

**Review by frankstepanski — CHANGES_REQUESTED** [review]
Need to update hard-coded server URL

**Review by frankstepanski — APPROVED** [review]
Good updates!

**Pull request merged**